checklist/app.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright 2020 The AMP HTML Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Application, Context} from 'probot';
+import Webhooks from '@octokit/webhooks';
+import {Octokit} from '@octokit/rest';
+
+import {ChecklistBot} from './src/checklist_bot';
+
+type WebhookPayload = Webhooks.WebhookPayloadPullRequest;
+
+module.exports = (app: Application) => {
+  if (process.env.NODE_ENV !== 'test') {
+    require('dotenv').config();
+  }
+
+  function botFromContext({github, payload, log}: Context<WebhookPayload>) {
+    return new ChecklistBot(
```

Comment:
Ryan created this because he needed to call this method multiple times. No reason why you can't just inline this in the single `app.on` handler that you have here :)

Reply:
I agree, but I didn't comment cause I knew you'd want to :)